partner_firstname/models/partner.py

```diff
@@ -1,26 +1,11 @@
 # -*- coding: utf-8 -*-
```

Comment:
That's why I suggest it instead of forcing, but I see it as a good standarization practise, so the name is not subjective to what the developer thinks that is the functional name.